src/formatting_stack/strategies/impl.clj

```diff
@@ -100,3 +100,10 @@
               (-> f .getCanonicalPath)))
        (some #{(-> file .getCanonicalPath)})
        (boolean)))
+
+(speced/defn ^boolean? git-ref-exists?
+  "Does `ref` denote an existing git branch, tag or commit sha?"
+  [^string? ref]
+  (-> (sh "git" "rev-parse" ref)
```

Comment:
`#'formatting-stack.strategies/git-command` is a constant for `"git"` (albeit you can't actually use it here)

Reply:
Interesting. `git-command` has no big purpose, simple it's something DRYed out of habit.

FWIW there's still no need of making the command configurable (and maybe it can be better configured by simply manipulating the PATH)

Reply:
Yes. I agreed. Might want to deprecate/remove that def at a later stage. But it's no issue rn